packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -91,6 +113,23 @@ const AppModals = ({
         ComponentToLoad = <RealityCheckModal />;
     }
 
+    if (
+        is_logged_in &&
+        active_account_landing_company === 'maltainvest' &&
+        !is_trading_assessment_for_new_user_enabled &&
+        cfd_score === 0
```

Comment:
If the user fails all the questions and gets the zero score, is this supposed to show the same component?

Reply:
For existing user, even if the user fails the assessment, it is not considered as a failure.